StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1612ElementParameterDocumentationMustMatchElementParameters.cs

```diff
@@ -36,11 +37,17 @@ public class SA1612ElementParameterDocumentationMustMatchElementParameters : Dia
         private const string Description = "The documentation describing the parameters to a C# method, constructor, delegate or indexer element does not match the actual parameters on the element.";
         private const string HelpLink = "http://www.stylecop.com/docs/SA1612.html";
 
-        private static readonly DiagnosticDescriptor Descriptor =
-            new DiagnosticDescriptor(DiagnosticId, Title, MessageFormat, Category, DiagnosticSeverity.Warning, AnalyzerConstants.EnabledByDefault, Description, HelpLink);
+        private const string MissingParamForDocumentationMessageFormat = "The parameter '{0}' does not exist.";
```

Comment:
Yes the warning is only for that case. The location of xxx is the location of the diagnostic, so I think the message should be okay. Or do you have a better message for it?

Im going to remove the unused constant